cmd/rekor-server/app/serve.go

```diff
@@ -235,6 +247,9 @@ func init() {
 	serveCmd.Flags().Uint("antispam-max-batch-size", 0, "maximum batch size for deduplication operations; will default to Tessera recommendation if unset; for Spanner, recommend around 1500 with 300 or more PU, or around 64 for smaller (e.g. 100 PU) instances")
 	serveCmd.Flags().Uint("antispam-pushback-threshold", 0, "maximum number of 'in-flight' add requests the antispam operator will allow before pushing back; will default to Tessera recommendation if unset")
 
+	// witness configs
+	serveCmd.Flags().String("witness-policy-file", "", "optional witness policy file")
```

Comment:
nit - would be more consistent with other flags to call this `witness-policy-path`, or more concise to just call it `witness-policy`.

Reply:
Looks like we have both `-path` and `-file`. I'll go with `path`.